dll/apisets/api-ms-win-appmodel-runtime-l1-1-1.spec

```diff
@@ -8,7 +8,7 @@
 @ stub GetCurrentApplicationUserModelId
 @ stub GetCurrentPackageFamilyName
 @ stub GetCurrentPackageFullName
-@ stub GetCurrentPackageId
+@ stdcall GetCurrentPackageId(ptr ptr) kernel32.GetCurrentPackageId
```

Comment:
I'm not convinced about that, because kernel32 does not have the API exported, so this redirection will fail anyways when code will call it.

Reply:
I tested it, and the export does indeed forward correctly (verified by looking for the STUB message in the debug log). If you're talking about the `version=0x602+` change Mark mentioned above, then I should adjust this spec file as well.